src/lib/arch/riscv/ExceptionHandler.cc

```diff
@@ -263,7 +255,7 @@ void ExceptionHandler::resetState() {
   // Reset state of handler
   instruction_ = nullptr;
   syscallReturned_ = false;
-  resumeHandling_ = [this]() { return initException(); };
+  envokingSycallHandler_ = false;
```

Comment:
Should this not be "invokingSyscallHandler_"? I've always assumed that you are invoking, but you can envoke/invoke. Very very minor so alter at your discression

Reply:
Got them mixed up, changing